src/Components/Home/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import { Typography } from '@mui/material';
+
+const HomeScreen = () => {
+  return (
+    <Box
+      sx={{
+        width: '99.9%',
+        textAlign: 'center',
+        minHeight: '100vh',
+        overflow: 'hidden',
+      }}>
+      {/* Slider component */}
+      <Box
+        sx={{
+          width: '100%',
+          objectFit: 'fill',
+          overflow: 'hidden',
+          maxHeight: '500px',
+        }}>
+        <img src="https://picsum.photos/3200/1500" width="100%" />
+      </Box>
+      <Container sx={{ marginTop: '20px' }}>
+        <Typography variant="h3">Texto de bienvenida</Typography>
+      </Container>
+      <Container sx={{ marginTop: '50px' }}>
+        <Box sx={{ objectFit: 'contain' }}>
+          <Typography variant="h4">Últimas novedades</Typography>
+          {/* UltimasNovedades Component */}
```

Comment:
please delete this comment. The code should be enougth documentation to itself. Add a div with a name="news"